src/test/app/AccountDelete_test.cpp

```diff
@@ -912,6 +912,275 @@ class AccountDelete_test : public beast::unit_test::suite
         env.close();
     }
 
+    void
+    testDestinationDepositAuthCredentials()
+    {
+        {
+            testcase(
+                "Destination Constraints with DepositAuth and Credentials");
+
+            using namespace test::jtx;
+
+            Account const alice{"alice"};
```

Comment:
Can you add a comment here about what each account's role is? e.g. Carol is the issuer, Becky is deleting her account, Alice is the destination.

Or rename them to be even clearer.

Reply:
Ditto for all the other tests.

Reply:
I'm going back and forth.

Reply:
Ye, but the same thing is true  for every other tests. Which of them source which is destination? What do zelda do in payment ? etc...